### PR TITLE
Update post-checkout Twitter share message copy

### LINKED ISSUE #2610: Explorer: Update post-checkout Twitter share message copy
**Overview**
Update copy in Twitter share message from [Thank You page](https://explorer.gitcoin.co/#/thankyou) to include tagging @grantsstack 

Updated copy: "I just donated to [Round Name] on @gitcoin @grantsstack. Join me in making a difference by donating today, and check out the projects I supported on my Donation History page!" 

**Details**
Say more

### PR DESCRIPTION
**Pull request by bhargavaparoksham**
## PR checklist

For every PR, make sure that these statements are true:
- [x] Includes only changes relevant to the original ticket. Significant refactoring needs to be separated.
- [x] Doesn't contain type casts and non-null assertions.
- [x] Doesn't add `@ts-ignore`.
- [x] Doesn't disable lints.
- [x] Doesn't use `useState` just for computation - use plain variables instead.
- [x] Splits components into pure components that don't depend on external state or hooks.
- [x] Avoid embedding components within other components
- [x] Doesn't propagate optional values without good reason, doesn't mark property values as optional if that doesn't represent reality.
- [x] Doesn't duplicate existing code.
- [x] Parses out-of-domain data - this includes user input, API respones, on-chain data etc.
- [x] Doesn't contain commented out code.
- [x] Doesn't contain skipped or empty tests.
- [x] If this PR adds/updates any feature, it adds/updates its test script

Subjective - at the discretion of the reviewers
- Does things as simply as possible, but not simpler.
- Doesn't reinvent the wheel or create premature abstractions.

##### Description

Explorer: Update post-checkout Twitter share message copy

##### Refers/Fixes

fixes #2610

### CODE REVIEW

**Review by carlbarrdahl — APPROVED** [review]

File: packages/grant-explorer/src/features/round/ThankYou.test.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ describe("createTwiterShareText", () => {
         isMrc: false,
       })
     ).toEqual(
-      `I just donated to a round on @gitcoin. Join me in making a difference by donating today, and check out the projects I supported on my Donation History page!
+      `I just donated to a round on @gitcoin @grantsstack. Join me in making a difference by donating today, and check out the projects I supported on my Donation History page!
```
Comment (bard) [inline packages/grant-explorer/src/features/round/ThankYou.test.tsx:69]: Should it be `@gitcoin's @grantsstack` (as in the text on https://twitter.com/grantsstack)?

**Review by bard — APPROVED** [review]

File: packages/grant-explorer/src/features/round/ThankYou.test.tsx (hunk 1 above)
Comment (bhargavaparoksham) [reply packages/grant-explorer/src/features/round/ThankYou.test.tsx:69]: @michellema1208

**Review by melissa-neira — CHANGES_REQUESTED** [review]
there seems to be an extra space after the round name
Screenshot 2023-11-16 at 14 29 46

File: packages/grant-explorer/src/features/round/ThankYou.test.tsx (hunk 1 above)
Comment (michellema1208) [reply packages/grant-explorer/src/features/round/ThankYou.test.tsx:69]: @bhargavaparoksham @bard good catch, let's do that!

**Review by melissa-neira — APPROVED** [review]
lgtm!

**Pull request merged**